syfertext/doc.py

```diff
@@ -446,11 +445,7 @@ def create_pointer(
             owner = doc.owner
 
         doc_pointer = DocPointer(
-            location=location,
-            id_at_location=id_at_location,
-            owner=owner,
-            id=ptr_id,
-            garbage_collect_data=garbage_collect_data,
+            location=location, id_at_location=id_at_location, owner=owner, id=ptr_id
```

Comment:
please add `garbage_collect_data = True`. Maybe I was not clear enough, excuse me. I wanted it to be removed only from the `__init__` function definition. Although, the super class have it as True by default, but I prefer we send it explicitely to super, to make sure that if PySyft guys changed the default, this wont' affect us

Reply:
The above is a constructor of `DocPointer` and if I add `garbage_collect_data = True` in it, then I can't remove it from ```__init__```, cause if I remove it from the constructor, I can't pass it while creating the object. :smile_cat: